<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTDenter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Timeline</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTreeWidget" name="timeline_list">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideRight</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="indentation">
           <number>0</number>
          </property>
          <property name="uniformRowHeights">
           <bool>false</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerDefaultSectionSize">
           <number>100</number>
          </attribute>
          <column>
           <property name="text">
            <string>Avatar</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Dent</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>dent_id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>denter_screen_name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>dent_text</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Mentions</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTreeWidget" name="mentions_list">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideRight</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="indentation">
           <number>0</number>
          </property>
          <property name="uniformRowHeights">
           <bool>false</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerDefaultSectionSize">
           <number>100</number>
          </attribute>
          <column>
           <property name="text">
            <string>Avatar</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Dent</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>dent_id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>denter_screen_name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>dent_text</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Direct messages</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTreeWidget" name="dm_list">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideRight</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="indentation">
           <number>0</number>
          </property>
          <property name="uniformRowHeights">
           <bool>false</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerDefaultSectionSize">
           <number>100</number>
          </attribute>
          <column>
           <property name="text">
            <string>Avatar</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Dent</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>dent_id</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>denter_screen_name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>dent_text</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Options"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Timelines">
    <property name="title">
     <string>&amp;Timelines</string>
    </property>
    <addaction name="action_Update_all"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionStatistics"/>
    <addaction name="separator"/>
    <addaction name="action_About_Denter"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Timelines"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_Options">
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Update_all">
   <property name="text">
    <string>&amp;Update all</string>
   </property>
  </action>
  <action name="action_About_Denter">
   <property name="text">
    <string>&amp;About QTDenter...</string>
   </property>
  </action>
  <action name="actionStatistics">
   <property name="text">
    <string>Statistics...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
